File: cfgs/pythoncfgs/DISTRIBUTING_ITEMS_PERSON_CANNOT_TAKE_TWO_ITEMS_TYPE.py
Nodes
1;conditional;fun_1(foo_1);8;
2;statement;return true;15;
3;statement;foo_1 = 0;9;
4;conditional;fun_1(foo_1);10;
5;conditional;foo_1[foo_2] == foo_3[foo_4];11;
6;conditional;foo_1 > 2 * foo_2;13;
7;statement;return false;14;
8;statement;foo_1 += 1;12;
9;exit;;
Edges
0;2;3;
1;9;
2;4;
3;1;5;
4;6;8;
5;4;7;
6;9;
7;6;
8;

File: cfgs/java_cfgs_foo/DISTRIBUTING_ITEMS_PERSON_CANNOT_TAKE_TWO_ITEMS_TYPE.java
Nodes
1;special;;;
2;statement;int count;14;
3;statement;int i = 0;15;
4;conditional;foo_1 < foo_2;16;
5;statement;count = 0;18;
6;statement;int j = 0;19;
7;conditional;foo_1 < foo_2;20;
8;conditional;foo_1[foo_2] == foo_3[foo_4];22;
9;statement;count++;22;
10;conditional;foo_1 > (2 * foo_2);23;
11;statement;return false;23;
12;exit;;;
13;statement;j++;21;
14;statement;i++;17;
15;statement;return true;26;
Edges
0;2;
1;3;
2;4;
3;5;15;
4;6;
5;7;
6;8;14;
7;9;10;
8;10;
9;11;13;
10;12;
11;
12;7;
13;4;
14;12;
